### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by badger-99**
In this Milestone, I made the following additions:

    A contact section with a form 
    A footer with social media links

### CODE REVIEW

**Review by ademibowale — COMMENTED** [review]
Hi @badger-99  ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request




Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

You have submitted a project,that non of your pair partner has a commit on your project please kindly go through the requirement and ensure  that all your partner has a commit msg on your projec, before you submit your project for review.please use this [Template](https://github.com/microverseinc/linters-config/tree/master/html-css-js) to create a professional documentation

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -299,6 +299,61 @@ <h3 class="skill_title">
...
+    <!-- Contact title -->
+    <h2 id="contact-title">
+      I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </h2>
+
+    <!-- Form -->
+    <form id="contact-form" action="https://formspree.io/f/xgebdkkr" method="post">
+      <input class="form-elm form-input" id="name" type="text" maxlength="30" placeholder="Full name" name="name" required>
+      <input class="form-elm form-input" id="email" type="email" placeholder="Email address" name="email" required>
+      <textarea class="form-elm" id="txt-area" name="message" cols="30" rows="10" maxlength="500" placeholder="Write me something..."></textarea>
```
Comment (LionRouge1) [inline index.html]: - [x] Nice work implementing the form, it looks really good :clap: However it's required that users cannot submit the form if one or more of the fields is empty (see image), so kindly add `required` attribute to the message field to fulfil that.
Screenshot from 2023-05-03 06-47-21

File: README.md
Comment (LionRouge1) [inline README.md:1]: - [x] Kindly update the "Setup" section in your README file to suit this project. For example, the repo link that is added is not the link of this repo, kindly check it to make your README file professional.

- [OPTIONAL] Kindly add your Linkedin and Twitter account to your README file. That will make it easier for visitors or recruiters to reach out to you.

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @badger-99, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good job your design look almost identical to the Figma design :100: 
- Good commit :+1: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joel-kalema — APPROVED** [review]
Hi Team

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**